Azure/azure-cli
# {Redis} Updating Redis api version in Private endpoint configuration

**Description**
Updating the Redis api version to the latest in private endpoint configuration

**Testing Guide**
Test private endpoint commands over redis resources
example: az network private-endpoint-connection list --id /subscriptions/{subscriptionID}/resourceGroups/{ResourceGroupName}/providers/Microsoft.Cache/Redis/{AzureCacheName}

This checklist is used to make sure that common guidelines for a pull request are followed.
